setuptools/_distutils/tests/test_bdist_rpm.py

```diff
@@ -77,11 +80,11 @@ def test_quiet(self):
         cmd.run()
 
         dist_created = os.listdir(os.path.join(pkg_dir, 'dist'))
-        self.assertIn('foo-0.1-1.noarch.rpm', dist_created)
```

Comment:
Can this change have its own unit test, rather than mutating an existing test?

Reply:
I've just now written up a dedicated test.